extensions/dicom-microscopy/src/utils/RoiAnnotation.js

```diff
@@ -0,0 +1,195 @@
+import Publisher from './Publisher';
+import areaOfPolygon from './areaOfPolygon';
+
+const EVENTS = {
+  LABEL_UPDATED: 'labelUpdated',
+  GRAPHIC_UPDATED: 'graphicUpdated',
+  VIEW_UPDATED: 'viewUpdated',
+  REMOVED: 'removed',
+};
+
+/**
+ * Represents a single annotation for the Microscopy Viewer
+ */
+class RoiAnnotation extends Publisher {
```

Comment:
same here, move all services inside a folder called services and use OHIF services manager